docs/sdks/client-sdks/flutter/bandits.mdx

```diff
@@ -0,0 +1,163 @@
+
+In order for the bandit to learn an optimized policy, we need to capture and log the bandit's actions.
+The Dart SDK uses a single logger class that handles both assignment and bandit action logging:
+
+```dart
+// Implement your own `BanditLogger`
+class YourBanditLogger implements BanditLogger {
+  @override
+  void logBanditEvent(BanditEvent event) {
+    // Your logging logic here
+  }
+}
+await Eppo.initialize(
+  PrecomputeArguments(subject: subject),
+  SdkOptions(
+    sdkKey: 'your-sdk-key',
+    banditLogger: YourBanditLogger(),
+  )
+);
+```
+
+The SDK will invoke the `logBanditAction()` method with a `BanditEvent` object containing the following fields:
+
+```dart
+/// Represents a bandit event.
+class BanditEvent {
+  /// The timestamp when the event occurred
+  final String timestamp;
+
+  /// The feature flag key
+  final String featureFlag;
+
+  /// The bandit key
+  final String bandit;
+
+  /// The subject key
+  final String subject;
+
+  /// The action associated with the bandit
+  final String? action;
+
+  /// The probability of the action
+  final double? actionProbability;
+
+  /// The optimality gap
+  final double? optimalityGap;
+
+  /// The model version
+  final String? modelVersion;
+
+  /// Numeric attributes of the subject
+  final Map<String, double>? subjectNumericAttributes;
+
+  /// Categorical attributes of the subject
+  final Map<String, String>? subjectCategoricalAttributes;
+
+  /// Numeric attributes of the action
+  final Map<String, double>? actionNumericAttributes;
+
+  /// Categorical attributes of the action
+  final Map<String, String>? actionCategoricalAttributes;
+
+  /// Metadata about the SDK
+  final Map<String, dynamic> metaData;
+}
+```
+
+Example of a `BanditEvent`:
+
+| Field                          | Example                          |
+| ------------------------------ | -------------------------------- |
+| `timestamp`                    | `"2024-03-22T14:26:55.000Z"`     |
+| `featureFlag`                  | `"bandit-test-allocation-4"`     |
+| `bandit`                       | `"ad-bandit-1"`                  |
+| `subject`                      | `"ed6f85019080"`                 |
+| `subjectNumericAttributes`     | `{"age": 30}`                    |
+| `subjectCategoricalAttributes` | `{"loyalty_tier": "gold"}`       |
+| `action`                       | `"promo-20%-off"`                |
+| `actionNumericAttributes`      | `{"brandAffinity": 0.2}`         |
+| `actionCategoricalAttributes`  | `{"previouslyPurchased": false}` |
+| `actionProbability`            | `0.25`                           |
+| `optimalityGap`                | `456`                            |
+| `modelVersion`                 | `"v123"`                         |
+| `metaData`                     | `{ "sdkLibVersion": "3.5.1" }`   |
+
+### Query the bandit for an action
+
+To query the bandit for an action, use the `getBanditAction()` method. This method takes the following parameters:
```

Comment:
We also need to provide the possible actions when initializing, this should probably be included in our additional steps section above